pkg/api/v1/schema.go

```diff
@@ -210,12 +210,33 @@ type CPU struct {
 	// Defaults to host-model.
 	// +optional
 	Model string `json:"model,omitempty"`
+	// Features specifies the CPU features list inside the VMI.
+	// For more information see https://libvirt.org/formatdomain.html#elementsCPU.
```

Comment:
we shouldn't be referencing libvirt documentation here. This is the kubevirt api. How our api maps to a libvirt domain is an implementation detail. 

It's confusing to have people learn about this feature by reading about libvirt. Libvirt doesn't explain how the KubeVirt feature impacts scheduling, or how this is related to the NFD component.  I see that libvirt documentation was referenced for CPU model as well. That shouldn't have made it in. We need to be documenting our API. How our API relates to Libvirt is irrelevant from the KubeVirt user's perspective.

Reply:
It was indeed taken and placed here according to the link we have in our user guide. 
ill remove the reference here and also in user guide.